bskyogcard/src/theme/index.ts

```diff
@@ -0,0 +1,109 @@
+import {colors} from './colors.js'
+import * as tokens from './tokens.js'
+
+export {atoms} from './atoms.js'
+export * as tokens from './tokens.js'
+
+const fontFamily = 'Inter'
+
+export const theme = {
+  name: 'light',
+  palette: colors,
+  atoms: {
+    text: {
+      color: colors.black,
+      fontFamily,
+    },
+    text_contrast_low: {
```

Comment:
Where does the snake_case convention come from throughout the theme config?  Is it something from the main social-app codebase?